Dapper.Contrib.MsSqlTests NET45/Program.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Data.SqlClient;
+using System.Data.SqlServerCe;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+
+namespace Dapper.Contrib.Tests
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            SetupMsSqlDatabase();
+            SetupTables();
+            RunTests();
+            DropTables();
+            SetupTables();
+            RunAsyncTests();
+            Console.WriteLine("Press any key...");
+            Console.ReadKey();
+        }
+
+        private static void SetupMsSqlDatabase()
+        {
+            using (var connection = new SqlConnection("Data Source = .\\SQLEXPRESS;Initial Catalog=master;Integrated Security=SSPI"))
+            {
+                connection.Open();
+                var exists = connection.Query<int>("SELECT count(*) FROM master.sys.databases WHERE name = @name",
+                    new { name = "DapperContribMsSqlTests" }).First();
+                if (exists > 0)
+                {
+                    connection.Execute("drop database DapperContribMsSqlTests");
+                }
+                connection.Execute("create database DapperContribMsSqlTests");
+
+            }
+        }
+
+        private static void DropTables()
+        {
+
+            using (var connection = new SqlConnection("Data Source = .\\SQLEXPRESS;Initial Catalog=DapperContribMsSqlTests;Integrated Security=SSPI"))
+            {
+                connection.Open();
+                connection.Execute("alter database DapperContribMsSqlTests set single_user with rollback immediate");
+                connection.Execute(@" drop table Stuff");

```

Comment:
Actually not sure what you mean Nick. SetupMsSqlDatabase(), SetupTables and DropTables() are part of the setup, not the tests.